_content/articles/buddhist-entrepreneurs-and-new-venture-performance_Liu-et-al.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Buddhist entrepreneurs and new venture performance: The mediating role of entrepreneurial risk-taking"
+authors:
+  - "Zhiyang Liu"
+  - "Zuhui Xu"
+  - "Zhao Zhou"
+  - "Yong Li"
+external_url: "https://papers.ssrn.com/sol3/Delivery.cfm/SSRN_ID3044525_code651404.pdf?abstractid=3044525&mirid=1"
+drive_links:
+  - "https://drive.google.com/file/d/1Ghsme80TosXsfQZM703US0BcI0g3x1rf/view?usp=drive_link"
+tags:
+  - business
```

Comment:
Yeah, the natural place for it would be "business" (or possibly `becon`?) I put it in the medieval course as an evolutionary argument about why Buddhism was so successful among medieval traders.